packages/components/src/components/file-upload/file-upload.tsx

```diff
@@ -33,8 +33,12 @@ export class FileUpload {
   @Prop() disabled: boolean = false;
   @Prop() maxFileSizeMB: number = 7;
   /** Default set of allowed file extensions (used internally). Can be extended using `additionalAllowedFileTypes`. */
-  @Prop() allowedFileTypes: string | string[] = ['jpg', 'jpeg', 'png', 'pdf', 'mov', 'mp3', 'mp4'];
+  @Prop() allowedFileTypes?: string | string[] = undefined;
```

Comment:
I think the undefined isnt needed, no? it should be the default for optional props